packages/mobile/scripts/pre-dev.sh

```diff
@@ -10,4 +10,7 @@ ENV_FILENAME="${ENVFILE:-.env}"
 export $(grep -v '^#' $ENV_FILENAME | xargs)
 echo "Building sdk for testnet $DEFAULT_TESTNET"
 yarn build:sdk $DEFAULT_TESTNET
-echo "Done building sdk"
\ No newline at end of file
+echo "Done building sdk"
+echo "Jetifying react native libraries"
+cd ../../ && yarn run jetify
```

Comment:
This is required to move some libs to AndroidX - namely rn-firebase, which we can hopefully update to v6 soon